MAINTAINERS.md

```diff
@@ -41,6 +41,8 @@ If you are an owner of the organization, you should be able to see an automated
   [05 December 2020](https://github.com/tldr-pages/tldr/issues/4994) — present
 - **bl-ue ([@bl-ue](https://github.com/bl-ue))**:
   [01 January 2021](https://github.com/tldr-pages/tldr/issues/5056) — present
+- **MasterOdin ([@MasterOdin](https://github.com/MasterOdin))**:
```

Comment:
Why not add your name, since it's displayed on your profile?
```suggestion
- **Matthew Peveler ([@MasterOdin](https://github.com/MasterOdin))**:
```

Reply:
Unless you'd prefer otherwise :)

Reply:
Because I copy pasted your line and then didn't really consider the other lines that had names. Whoops.